libs/colbert/ragstack_colbert/base_retriever.py

```diff
@@ -0,0 +1,48 @@
+"""
```

Comment:
changed all the interfaces to be prefixed with `base_` and renamed the class name too.